x/blob/client/cli/payforblob.go

```diff
@@ -57,10 +60,24 @@ func CmdPayForBlob() *cobra.Command {
 	}
 
 	flags.AddTxFlagsToCmd(cmd)
+	cmd.PersistentFlags().String(FlagNamespaceVersion, "0", "Specify the namespace version (default is 0)")
```

Comment:
we shouldn't write defaults, since they're filled in for us by cobra
```suggestion
	cmd.PersistentFlags().String(FlagNamespaceVersion, "0", "Specify the namespace version")
```

Reply:
@evan-forbes where are they filled in? After this suggestion the help doc looks like:

```
$ ./build/celestia-appd tx blob PayForBlobs --help
...
      --namespace-version uint8   Specify the namespace version
```

Reply:
hmmm I'm not sure that's a good question. the rest get filled in for example https://github.com/celestiaorg/cosmos-sdk/blob/f19e20459b8bcf195167ad256de5a6ca6edc41f5/server/start.go#L159-L161

```
--address string                                  Listen address (default "tcp://0.0.0.0:26658")
--home string                                     The application home directory (default "/home/evan/.celestia-app")
```